bin/circleci/build-addon.sh

```diff
@@ -2,4 +2,3 @@
 set -ex
 cd addon/
 npm run package
-cp *.rdf *.xpi $CIRCLE_ARTIFACTS/
```

Comment:
Why did you remove this? It would be useful to have the addon in CIRCLE_ARTIFACTS for debugging stuff.

Reply:
Is the default directory the artifacts dir now?

Reply:
Yeah I added it back. Its in the ```circleci.yml```. If you look at the ```build``` step you can see the artifacts.

Reply:
Artifacts: https://circleci.com/docs/2.0/artifacts/#uploading-artifacts